DeviceAdministration/Infrastructure.UnitTests/Common/DeviceSchemaHelperTests.cs

```diff
@@ -162,6 +162,75 @@ public void GetHubEnabledStateShouldReturnNullButNotThrowIfMissingState()
 
         #endregion
 
+        #region IotHub tests
+
+        [Fact]
+        public void getIotHubPropertiesTest()
+        {
+            DeviceModel d = GetValidDevice();
+            IoTHub props = DeviceSchemaHelper.GetIoTHubProperties(d);
+            string messageId = props.MessageId;
+
+            Assert.Equal("messageId", messageId);
+        }
+
+        [Fact]
+        public void missingIotHubPropertiesTest()
+        {
+            DeviceModel d = GetDeviceWithMissingDeviceProperties();
+
```

Comment:
I use resharper to cleanup the code: ctrl+alt+f
it'll delete the whitespaces and new lines.